contacts.module

```diff
@@ -85,7 +86,40 @@ function template_preprocess_contacts_dash_tabs(array &$variables) {
  *   - region_attributes: HTML markup attributes for the content wrapper.
  */
 function template_preprocess_contacts_dash_tab_content(array &$variables) {
+  $variables['messages'] = $variables['content']['messages'];
   $variables['region_attributes'] = new Attribute($variables['region_attributes']);
+  /* @var \Drupal\contacts\Entity\ContactTab $tab */
+  $tab = \Drupal::service('contacts.tab_manager')->getTabByPath($variables['user'], $variables['subpage']);
+  $layout = $tab->get('layout') ?: 'contacts_tab_content.stacked';
+
+  // @todo switch to core layout manager plugin.manager.core.layout?
+  /* @var \Drupal\layout_plugin\Plugin\Layout\LayoutPluginManager $layoutPluginManager */
+  $layoutPluginManager = \Drupal::service('plugin.manager.layout_plugin');
+  /* @var \Drupal\layout_plugin\Plugin\Layout\LayoutInterface $layoutInstance */
+  $layoutInstance = $layoutPluginManager->createInstance($layout, []);
+
+  $regions = [];
+
+  foreach ($layoutInstance->getPluginDefinition()['regions'] as $key => $region) {
+    if (!empty($variables['content'][$key])) {
+      $regions[$key] = $variables['content'][$key];
```

Comment:
Mentioned in #63, but we should definitely move building the block render array into here and probably move it into a render element for OO bliss...